ansible/roles/kubernetes/tasks/gen_tokens.yml

```diff
@@ -4,27 +4,35 @@
     src=kube-gen-token.sh
     dest={{ kube_script_dir }}
     mode=u+x
+  when: inventory_hostname == groups['masters'][0]
+  run_once: true
 
 - name: Generate tokens for master components
   command: "{{ kube_script_dir }}/kube-gen-token.sh {{ item[0] }}-{{ item[1] }}"
   environment:
     TOKEN_DIR: "{{ kube_token_dir }}"
+  delegate_to: "{{ groups['masters'][0] }}"
+  run_once: true
+  when: inventory_hostname == groups['masters'][0]
```

Comment:
Did other masters except master[0] will get an empty ( skipped value ) register variable of  `gentoken_master` and `gentoken_node`?